logic.php

```diff
@@ -4035,3 +4039,89 @@ function curl_json_response($json_response, $json)
     return $response;
 }
 
+function alarm($raid,$user,$action,$info = '')
```

Comment:
Can you add the usual scheme above all 3 new functions to describe the function, list the vars and return?